include/bmi_cfe.h

```diff
@@ -5,20 +5,160 @@
 extern "C" {
 #endif
 
+#include "../original_author_code/cfe.h"
```

Comment:
Why reference this original_author_code folder directly?  Better to update [cfe.h](https://github.com/jmframe/cfe/blob/bmi_from_driver/include/cfe.h) and include this header path to avoid confusion.

Reply:
My thinking was that this would be a directory with code that has been approved by Fred. It doesn't matter to me though. I'll move it.